From dc3fb86d295d2cdfc11f6d0687c1b7eff8d43b94 Mon Sep 17 00:00:00 2001
From: Charles Bousseau <cbousseau@anaconda.com>
Date: Thu, 25 May 2023 17:56:53 -0400
Subject: [PATCH 19/24] Remove unused readelf

readelf has been dropped.
.. date: 2022-11-02-10-56-40
.. gh-issue: 98949
.. nonce: 3SRD8C
.. section: Build

Drop unused build dependency on ``readelf``.
---
 Makefile.pre.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index 66b34b779f2..991499ee456 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -38,7 +38,6 @@ CC=		@CC@
 CXX=		@CXX@
 LINKCC=		@LINKCC@
 AR=		@AR@
-READELF=	@READELF@
 SOABI=		@SOABI@
 ABIFLAGS=	@ABIFLAGS@
 ABI_THREAD=	@ABI_THREAD@
